packages/mint-components/src/components/tax-and-cash/sqm-indirect-tax-form/sqm-indirect-tax-form.feature

```diff
@@ -0,0 +1,81 @@
+@owner:andy @author:andy
+Feature: Indirect Tax Form
+
+  Background: A user has submitted their personal information in Tax Form Step One
+    Given a user is on the Indirect Tax Form
+
```

Comment:
@Locrian24 These specs need to include what happens when you add the information and actually submit